js/client-rooms.js

```diff
@@ -118,14 +118,16 @@
 		title: 'Battles',
 		isSideRoom: true,
 		events: {
-			'change select[name=elofilter]': 'refresh'
+			'change select[name=elofilter]': 'refresh',
+			'click .search': 'refresh'
```

Comment:
Not sure if `.search` is the best choice (is there a more precise way of accessing this node in the DOM like with `select[name=elofilter]`?